android/assets/jsons/translations/French.properties

```diff
@@ -1474,11 +1474,11 @@ Gain a free [building] [cityFilter] =
 
 Only available after [] turns = Seulement disponible après [] tours
  # Requires translation!
-This Unit upgrades for free = 
+This Unit upgrades for free = Cette unité s'est améliorée gratuitement
```

Comment:
Isn't the left one talking about a not yet realized possibility while your translation sounds past tense?

Reply:
I thought it was a summary of the precedent turn, sorry.

Reply:
Nonono I formulated that as question because I did not actually check the context... OK checking now... Actually that's used for Ancient Ruins rewards. Those are instantaneous, so present/past might be better? But then, I _think_ you see that text only in the Civilopedia listing of possible ruin rewards... Where the potential version would look better?

Hmmm, the Pedia entry does use different texts altogether??? Sorry, then I don't know the context where this string is seen. (OK, this exact one is commented as `// Not used in Vanilla`, but its sibling `"This Unit upgrades for free including special upgrades"` should be seen somewhere)